exporter/dorisexporter/exporter_common.go

```diff
@@ -0,0 +1,109 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package dorisexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter"
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql" // for register database driver
```

Comment:
Is that the only type of DB supported?

Reply:
In fact, we use mysql's dirver to connect to doris instead of mysql.